choreolib/src/main/java/choreo/Choreo.java

```diff
@@ -335,124 +324,12 @@ public void clear() {
-   *     be called every loop during the command.
-   * @param bindings Universal trajectory event bindings.
-   * @param trajectoryLogger A {@link TrajectoryLogger} to log {@link Trajectory} as they start and
-   *     finish.
-   * @return An {@link AutoFactory} that can be used to create {@link AutoRoutine} and {@link
-   *     AutoTrajectory}.
-   * @see AutoChooser using this factory with AutoChooser to generate auto routines.
-   */
-  public static <SampleType extends TrajectorySample<SampleType>> AutoFactory createAutoFactory(
-      Supplier<Pose2d> poseSupplier,
-      Consumer<Pose2d> resetOdometry,
-      Consumer<SampleType> controller,
-      BooleanSupplier useAllianceFlipping,
-      Subsystem driveSubsystem,
-      AutoBindings bindings,
-      TrajectoryLogger<SampleType> trajectoryLogger) {
-    return new AutoFactory(
-        requireNonNullParam(poseSupplier, "poseSupplier", "Choreo.createAutoFactory"),
-        requireNonNullParam(resetOdometry, "resetOdometry", "Choreo.createAutoFactory"),
-        requireNonNullParam(controller, "controller", "Choreo.createAutoFactory"),
-        requireNonNullParam(driveSubsystem, "driveSubsystem", "Choreo.createAutoFactory"),
-        requireNonNullParam(useAllianceFlipping, "useAllianceFlipping", "Choreo.createAutoFactory"),
-        requireNonNullParam(bindings, "bindings", "Choreo.createAutoFactory"),
-        Optional.of(trajectoryLogger));
-  }
-
-  /**
-   * Create a factory that can be used to create {@link AutoRoutine} and {@link AutoTrajectory}.
-   *
-   * @param <SampleType> The type of samples in the trajectory.
-   * @param poseSupplier A function that returns the current field-relative {@link Pose2d} of the
-   *     robot.
-   * @param resetOdometry A function that receives a field-relative {@link Pose2d} to reset the
-   *     robot's odometry to.
-   * @param controller A function that receives the current {@link SampleType} and controls the
-   *     robot.
-   * @param driveSubsystem The drive {@link Subsystem} to require for {@link AutoTrajectory} {@link
-   *     Command}s.
-   * @param useAllianceFlipping If this returns true, when on the red alliance, the path will be
-   *     mirrored to the opposite side, while keeping the same coordinate system origin. This will
-   *     be called every loop during the command.
-   * @param bindings Universal trajectory event bindings.
-   * @param trajectoryLogger A {@link TrajectoryLogger} to log {@link Trajectory} as they start and
-   *     finish.
-   * @param alliance A custom supplier of the current alliance to use instead of {@link
-   *     DriverStation#getAlliance}.
-   * @return An {@link AutoFactory} that can be used to create {@link AutoRoutine} and {@link
-   *     AutoTrajectory}.
-   * @see AutoChooser using this factory with AutoChooser to generate auto routines.
+   * @Deprecated Use ```new AutoFactory()``` instead.
```

Comment:
javadoc requires all the fields to be here, so you can't just delete all of them. Also, javadoc uses `@deprecated` instead of `@Deprecated`.